ci/playbooks/meta_content_provider/meta_content_provider.yml

```diff
@@ -47,7 +47,7 @@
     - name: Build openstack services container when gating repo exists
       when:
         - "'os-net-config' not in zuul_change_list"
-        - _gating_repo.stat.exists
+        - _gating_repo.stat.exists or force_container_build | default('false')
```

Comment:
This flag is nowhere described.
```
dasm@pc ci-framework % grep -rn force_container_build *
dasm@pc ci-framework %
```

This flag requires _at least_ explanation and description. It's not following `ci-framework` standards of describing values, simlar to `cifmw_*`

Reply:
Done!